packages/react/src/utils.ts

```diff
@@ -1,16 +1,16 @@
 import { invariant } from '@lens-protocol/shared-kernel';
-import { providers, Signer } from 'ethers';
+import { Signer } from 'ethers';
 
 import { RequiredSigner } from './wallet/adapters/ConcreteWallet';
 
 export function isRequiredSigner(signer: Signer): signer is RequiredSigner {
-  return '_signTypedData' in signer && signer.provider instanceof providers.JsonRpcProvider;
+  return '_signTypedData' in signer;
 }
 
 export function assertRequiredSigner(signer: Signer): asserts signer is RequiredSigner {
```

Comment:
Seems the `assertRequiredSigner` can be removed from the codebase (and from relevant docs)?